src/Components/ProductReviews/ProductReviews.css

```diff
@@ -0,0 +1,91 @@
+.review__input--container{
+    width: 100%;
+    display: flex;
+    justify-content: center;
+    flex-wrap: wrap;
+}
+
+.review__input{
+    padding: var(--padding-small);
+    font-size: var(--text-medium);
+    border: none;
+    border-bottom: 1px solid var(--clr-grey);
+    width: min(35rem,50%);
+}
+
+.review__input:focus{
+    outline: none;
+}
+
+.para-rating__container{
+    padding: 0.25rem 0.5rem;
+    font-size: 1.5rem;
+    background-color: green;
+    color: white;
```

Comment:
can use variables for colours
if possible try using different shades of the default colours, default colours are a little strong for eyes

Reply:
thanks, have updated the same